product-cli/cmd/nodepool/kubevirt/create.go

```diff
@@ -0,0 +1,33 @@
+package kubevirt
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/hypershift/cmd/nodepool/core"
+	hyperShiftKubeVirt "github.com/openshift/hypershift/cmd/nodepool/kubevirt"
+)
+
+func NewCreateCommand(coreOpts *core.CreateNodePoolOptions) *cobra.Command {
+	platformOpts := &hyperShiftKubeVirt.KubevirtPlatformCreateOptions{
+		Memory:             "4Gi",
```

Comment:
this should default to 8Gi. It was a mistake that we left it as `4Gi` for the dev nodepool cli command.